src/main/java/org/ibp/api/java/impl/middleware/dataset/AbstractDatasetExportService.java

```diff
@@ -79,6 +88,14 @@ File generate(
 
 		// Get all variables for the dataset
 		final List<MeasurementVariable> columns = this.getColumns(study.getId(), dataSet.getDatasetId());
+		if (dataSet.getDatasetTypeId().equals(DatasetTypeEnum.PLOT_DATA.getId())) {
+			final LotsSearchDto lotsSearchDto = new LotsSearchDto();
+			lotsSearchDto.setPlantingStudyIds(Arrays.asList(studyId));
```

Comment:
@cuenyad  Is it filtering the by cancelled transactions?

Reply:
Fixied